popgen/stationarity.go

```diff
@@ -164,8 +164,8 @@ func AFSSampleDensityOld(n int, k int, alpha float64, binomMap [][]float64) floa
 //AlleleFrequencyProbability returns the probability of observing i out of n alleles from a stationarity distribution with selection parameter alpha.
 func AlleleFrequencyProbability(i int, n int, alpha float64, binomMap [][]float64) float64 {
 	var denominator float64 = math.Inf(-1)//denominator begins at -Inf when in log space
-	//check if n has already been seen
-	for j := 1; j < n-1; j++ {
+	// j loops over all possible values of i
+	for j := 1; j < n; j++ {
```

Comment:
Yes this was simply an error from the (N-1) symbolic representation, which is inclusive (math vs CS style range)

Reply:
I don't have an intuitive sense of our our omission of the last integral (when j == n-1) will impact the overall behavior. Something we may just have to get an idea of empirically